guides/Compiling.md

```diff
@@ -99,6 +99,14 @@ The Android command line tools can be found on the Android website or installed
 manager.  The command line tools contain a tool named `sdkmanager` that can be used to install
 various versions of Android, the Android build tools, and the NDK.
 
+Installation via `sdkmanager` command line tool:
+
+    $ cmdline-tools/bin/sdkmanager --sdk_root=/path/to/Android/Sdk "build-tools;30.0.3" "cmake;3.22.1" "ndk;21.4.7075529" "platform-tools" "platforms;android-26
```

Comment:
Is `--sdk_root` needed?  Seems like it defaults to the SDK of the tool, and I've never had to add it.  One less thing to set up and fill out when trying to get started.

Reply:
Also maybe `ndk-bundle` so the latest gets installed instead of a specific version?  That's a minor thing though.

Reply:
FWIW I haven't needed to install/use `cmake` with sdkmanager, but I've heard from others that it's needed so maybe good to leave it in.

Reply:
sdk_root allows to choice Sdk installation path, for me it didn't work without path.

IMHO, it's better to leave specific version, at least it's tested with LOVR build. Update man if neccessary.

Cmake is mandatory, toolchain file are shipped inside cmake package.